sdk/openai/Azure.AI.OpenAI/src/Custom/ChatCompletions/ChatCompletionsOptions.cs

```diff
@@ -137,4 +137,16 @@ public ChatCompletionsOptions()
         Enhancements = null;
         Tools = new ChangeTrackingList<ChatCompletionsToolDefinition>();
     }
+
+    /// <summary>
```

Comment:
Question: Since `functions` and `function_call` are deprecated as per OpenAI's API reference, should we "hide" them using `[EditorBrowsable(EditorBrowsableState.Never)]`? Doing so could help reduce confusion and guide users naturally towards the new `tools` and `tool_choice`.

Reply:
We should do that as soon as we can fully confirm that *all* the extant deployable models can use `tools`. And I hope we can do that soon!